tests/Doctrine/DBAL/BatchedConnectionTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Tests\AppBundle\Doctrine\DBAL;
+
+use AppBundle\Doctrine\DBAL\BatchedConnection;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Statement;
+use PHPUnit\Framework\TestCase;
+
+class BatchedConnectionTest extends TestCase
+{
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|Connection
+     */
+    private $connection;
+
+    /**
+     * @var BatchedConnection
+     */
+    private $batchedConnection;
+
+    protected function setUp()
```

Comment:
Please implement `tearDown()` method and clean properties to avoid memory leaks between tests